automation_tools/__init__.py

```diff
@@ -218,13 +217,6 @@ def setup_default_docker():
     # https://github.com/docker/docker/issues/2758
     manage_daemon('restart', 'docker', pty=(os_version >= 7))
 
-    # Check that things look good
-    run('docker ps')
```

Comment:
@sthirugn as the commit message states, manage_daemon will fail if docker is not installed.